utils/docker/pull-or-rebuild-image.sh

```diff
@@ -48,14 +49,23 @@ function build_the_image() {
 	popd
 }
 
-# If "rebuild" param is passed to the script, force rebuild
+function pull_the_image() {
```

Comment:
Maybe it's me, but word "the" in function name looks odd.

Reply:
Done.